src/Cache/Value.php

```diff
@@ -25,6 +25,8 @@ class Value
 
     /**
      * the number of minutes until the value expires
+     * @todo Rename this property to $expiry to reflect
```

Comment:
Can't we do this directly? Since it's a protected property

Reply:
For consistency it would also require renaming the `$minutes` argument in the constructor, which would be a breaking-change in PHP 8 (named arguments). That's why I left it out for now.